core/src/main/resources/com/devonfw/tools/solicitor/config/solicitor_base.cfg

```diff
@@ -26,8 +26,8 @@
     "templateSource" : "classpath:com/devonfw/tools/solicitor/rules/rule_templates/LicenseAssignment.drt",
     "ruleGroup" : "LicenseAssignmentProject",
     "description" : "setting license in case that no one was detected or overwriting a wrongly detected license (project overrides; 'old' rule structure)",
-    "deprecationWarnOnly" : true,
-    "deprecationDetails" : "Use of decision table LicenseAssignmentProject is deprecated, use LicenseAssignmentV2Project instead. See release notes of release 1.4.0 for migration hints"
+    "deprecationWarnOnly" : false,
+    "deprecationDetails" : "Use of decision table LicenseAssignmentProject is deprecated, use LicenseAssignmentV2Project instead. See https://github.com/devonfw/solicitor/issues/263"
```

Comment:
We should list both tickets as the first gives some background and the second indicates that it is now stage 2.